wiki/Community/Mappers_Guild/en.md

```diff
@@ -71,3 +102,16 @@ Following in the footsteps of both of these beatmap packs, the Mappers' Guild co
 While the initial plan for the Mappers' Guild had always been to have MMO elements like quests, that was found to be impractical to handle while manually managing deadlines for the limited number of guild participants. ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) and ::{ flag=CL }:: [Milan-](https://osu.ppy.sh/users/1052994) began development of a website to automate Mappers' Guild at the end of 2018, allowing for more adequate distribution of rewards and management of quest/party systems.
 
 The [Mappers' Guild website](https://mappersguild.com/) launched on the 1st of March 2019, and is where everything about the Mappers' Guild is currently controlled. Since being opened to the public, the guild has reached a member count over 500 and publishes [update news posts](https://osu.ppy.sh/home/news/2019-11-29-featured-artist-beatmap-updates-from-the-mappers-guild) to keep the rest of the osu! community up-to-date with Featured Artist mapping content.
+
+### Notes
+
+[^party]: A group of mappers who collaborate on a quest together.
+[^price]: Amount of points required to accept the quest. These points are pulled from a user's "available points" pool, not their total earned points.
+[^points]: Rewards are based on a user's total points. A user's available points are a subsection of their total points following the same model as osu!'s [kudosu](/wiki/Modding/Kudosu) "total earned" and "available" pools.
+[^map]: Points for creating beatmaps scale according to a map's length, meaning a longer map rewards more points than a shorter map. Collaborative difficulties split points evenly between all mappers. Points are only rewarded to Ranked beatmaps.
+[^bonus]: Beatmaps assigned to a [quest](#quests) and [showcase beatmaps](/wiki/People/Featured_Artists/Featured_Artist_showcase_beatmaps) earn +2 bonus points per difficulty. If a map is ranked after the quest's deadline, this bonus will be reduced to 1.5 points for 0-20 days late, 1 point for 21-40 days late, or 0.5 points for 40+ days late. Winning beatmaps for [priority quests](#priority-quests) earn +3 bonus points per difficulty. Each of these bonuses scale by a map's length.
+[^mod]: Modding points for [Beatmap Nominators](/wiki/People/Beatmap_Nominators) who nominate a relevant beatmap scale with length because it's assumed that they mod all difficulties.
+[^storyboard]: Storyboard points are rewarded based on a storyboard's content. Minimalist storyboards (basic lyrics usually) reward 2 points, while detailed storyboards earn 10 points.
+[^priority-quest]: Priority quest points are based on the quest's tier. Tier 1 (easiest and most common) provides 5, tier 2 provides 8, tier 3 provides 5, and tier 4 provides 24 (hardest and least common).
```

Comment:
you may have made a typo for tier 3. did you mean 15?
tier 1, 2, 3, 4 --> 5, 8, *5*, 24 points